site/content/en/docs/Guides/Client SDKs/local.md

```diff
@@ -158,6 +158,34 @@ chmod o+r gameserver.yaml
 docker run --network=host --rm -v $(pwd)/gameserver.yaml:/tmp/gameserver.yaml us-docker.pkg.dev/agones-images/release/agones-sdk:{{<release-version>}} --local -f /tmp/gameserver.yaml
 ```
 
+If you would like to run the ClientSDK and your game server container together with Docker Compose create a `docker-compose.yaml` file.
+
+```yaml
+version: '3'
+services:
+  gameserver:
+    build: . # <location to your image here>
```

Comment:
Is `build` a path to the image, or a path to the build context? 

(Just looking at https://docs.docker.com/compose/compose-file/compose-file-v3/#build ?)

Reply:
Ah, good question. `build` is to the context, but also defaults to assuming `Dockerfile` is at the root of that path. Happy to split this out to make more explicit for the docs like what is shown in that second example in your link. What do you think?

Reply:
Updated verbiage here to `# <path to build context>`